docs/contributor-guide/getting-started.md

```diff
@@ -149,6 +184,17 @@ The build relies on the following assumptions:
 - All rules in `Makefile` should assume the working directory is `ofrak_package_x` (but at a different path as explained above)
 - `Dockerstub` and `Dockerstage` should be written assuming the build context is the parent directory of `ofrak_package_x`. Do not assume anything is present in the build context besides the contents of `ofrak_package_x` and what `Makefile` adds to `ofrak_package_x` in the `dependencies` rule.
 
+## Static Code Analysis
+The repository has several automated static code anaysis workflows.
+
+1. [Pre-commit](#pre-commit) is used, both as a pre-commit hook and as a CI/CD workflow, to ensure repository-wide:
+   1. [PEP8 compliance](#pep-8) (this is done using [black](#black)).
+   1. No Python modules contain unused imports.
+   1. Files are either empty or end with a newline.
+1. Each package contains an `inspect` target responsible for static code analysis for that package.
+   1. As part of this, [MyPy](#type-annotations-and-mypy) is used to perform static type checking.
```

Comment:
I am not 100% sure, but I vaguely remember that MkDocs does not allow numbered lists with the numbers all set to `1.`.

If I'm wrong, feel free to resolve this issue and re-request review, and I'll approve.

Reply:
I just reformatted it, but it seems to work:
<img width="757" alt="Screenshot 2023-04-25 at 11 29 09 AM" src="https://user-images.githubusercontent.com/53830972/234327189-8b4edaa2-c3a2-4e6c-97f8-a8778fba8568.png">